core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -361,7 +362,7 @@ object ConsoleConsumer extends Logging {
       else if (fromBeginning) OffsetRequest.EarliestTime
       else OffsetRequest.LatestTime
 
-    CommandLineUtils.checkRequiredArgs(parser, options, if (useNewConsumer) bootstrapServerOpt else zkConnectOpt)
+    CommandLineUtils.checkRequiredArgs(parser, options, if (useOldConsumer) zkConnectOpt else bootstrapServerOpt)
```

Comment:
We don't need to check `zkConnectOpt` as that is how `useOldConsumer` is defined.